<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="always_explore" default="false" />
    <arg name="agent" />

    <param name="/neuroracer_gym/always_explore" value="$(arg always_explore)" type="boolean"/>
    <param name="/neuroracer_gym/agent_name" value="$(arg agent)" type="string"/>
    
    <!-- This version uses the openai_ros environments -->
    <!-- <rosparam command="load" file="$(find neuroracer_gym_rl)/config/params.yaml" /> -->
    <!-- Launch the training system -->
    
    <node pkg="neuroracer_gym_rl" name="neuroracer_gym_rl" type="training.py" output="screen"/>
</launch>
